src/components/common/ClusterNotReachableDailog/ClusterNotReachableDialog.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { ConfirmationDialog } from '@devtron-labs/devtron-fe-common-lib'
+import warningIconSrc from '../../../assets/icons/ic-warning-y5.svg'
+import { BUTTON_TEXT, NONCASCADE_DELETE_DIALOG_INTERNAL_MESSAGE } from '../../../config/constantMessaging'
+
+interface ClusrerNotReachableDialogType {
+    clusterName: string
+    onClickCancel: () => void
+    onClickDelete: () => void
+}
+function ClusrerNotReachableDialog({ clusterName, onClickCancel, onClickDelete }: ClusrerNotReachableDialogType) {
+    const clusterNameString:string = clusterName ? ` '${clusterName}'` : ''
+    return (
+        <ConfirmationDialog>
+            <ConfirmationDialog.Icon src={warningIconSrc} />
+            <ConfirmationDialog.Body title={`The cluster${clusterNameString} is not reachable`} />
```

Comment:
Should there be space after cluster?

Reply:
I handled the space in clusterNameString itself. clusterNameString : ` Cluster Name` || `NULL String`